pysm/models/hd2017.py

```diff
@@ -235,16 +239,14 @@ def __init__(
                 ),
                 nside_out=nside,
             )
-        elif not rnd_uval:
+        else:
             # I think this needs filling in for case when ISRF is not
             # a random realization. What should the default be? Could
             # choose a single value corresponding to T=20K, beta_d=1.54?
-            pass
```

Comment:
@brandonshensley @bthorne93 I have implemented the `d8` model where instead of using `draw_uval` we set it `uval` to 0.2,
tests pass fine against PySM 2 results.
Based on @bthorne93 comment up here, do you think I should modify anything?

Reply:
uval = 0.2 is the intended default for the high-latitude sky and should correspond reasonably well to an MBB model with Td=20K and beta_d=1.54.

Reply:
thanks @brandonshensley, based on this: https://github.com/healpy/pysm/pull/51/commits/e3f6678e443a23a9c1c6a1b3fd1fbdeebb1e0f79